app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def render_if(condition, record)
+    if condition
+      render record
+    end
```

Comment:
Room to simplify this if you want
```suggestion
    render record if condition
```

Reply:
yush good call ty